app/helpers/marking_schemes_helper.rb

```diff
@@ -0,0 +1,53 @@
+module MarkingSchemesHelper
+  def get_table_json_data
```

Comment:
Assignment Branch Condition size for get_table_json_data is too high. [15.13/15]